configure.ac

```diff
@@ -2273,6 +2273,14 @@ then
         AC_MSG_ERROR([cannot enable eccencrypt without enabling hkdf.])
     fi
     AM_CFLAGS="$AM_CFLAGS -DHAVE_ECC_ENCRYPT"
+    if test "$ENABLED_ECC_ENCRYPT" = "old"
+    then
+        AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_ECIES_OLD"
```

Comment:
For existing users of the ECC encrypt feature they will have to add `WOLFSSL_ECIES_OLD`. Should we consider changing this so the SEC.1 mode is a new define? `DWOLFSSL_ECIES_SEC1` and `--enable-eccencrypt=sec1`?

Reply:
Todd said that he wanted the SEC1/ISO18033 to be default.